source/shared/cpp/ObjectModel/MarkDownParsedResult.cpp

```diff
@@ -0,0 +1,227 @@
+#include "MarkDownParsedResult.h"
+
+using namespace AdaptiveCards;
+
+void MarkDownParsedResult::Translate()
+{
+    MatchLeftAndRightEmphasises();
+}
+
+// appends html block tags at head and tail of the list
+void MarkDownParsedResult::AddBlockTags()
+{
+    // if html list tag, add <ul>, otherwise <p> at the head of the list
+    std::string tags = (m_codeGenTokens.front()->IsList())? "<ul>" : "<p>";
```

Comment:
><ul> [](start = 61, length = 4)

Do we need to do anything for ordered lists?

Reply:
yes, it is being worked on it.

---
In reply to: [157629049](https://github.com/Microsoft/AdaptiveCards/pull/1045#discussion_r157629049) [](ancestors = 157629049)